src/main/java/seedu/exercise/model/schedule/UniqueScheduleList.java

```diff
@@ -1,112 +1,19 @@
 package seedu.exercise.model.schedule;
 
-import static java.util.Objects.requireNonNull;
-import static seedu.exercise.commons.util.CollectionUtil.requireAllNonNull;
-
-import java.util.Iterator;
-import java.util.List;
-
-import javafx.collections.FXCollections;
-import javafx.collections.ObservableList;
-import seedu.exercise.model.schedule.exceptions.DuplicateScheduleException;
-import seedu.exercise.model.schedule.exceptions.ScheduleNotFoundException;
+import seedu.exercise.model.Resource;
+import seedu.exercise.model.UniqueList;
 
 /**
  * A list of schedules that enforces uniqueness between its elements and does not allow nulls.
  * A schedule is considered unique by comparing using {@code Schedule#isSameSchedule(Schedule)}.
  * As such, adding and updating of schedules uses Schedule#isSameSchedule(Schedule) for equality so as to ensure that
  * the schedule being added does not have any potential scheduling conflicts.
- *
+ * <p>
```

Comment:
Closing tag. \</p>

Reply:
File removed too with reasons similar to ``UniqueExerciseList``.